### DISCUSSION TITLE
🐻 v3.5.x

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#4522] [Releases]
The **release manager** for `v3.5.x` is @ESadek-MO

The **release buddy** for `v3.5.x` is @trexfeathers 

See the [v3.5.0 Project](https://github.com/orgs/SciTools/projects/4) for further details.

| Minor Release | Status | Project | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.5.0rc0` | Released | [Iris v3.5.0](https://github.com/orgs/SciTools/projects/4/views/1) | `14-04-2023` | [Docs](https://scitools-iris.readthedocs.io/en/v3.5.0rc0/) | [Release](https://github.com/SciTools/iris/releases/tag/v3.5.0rc0) | [conda-forge](https://anaconda.org/conda-forge/iris?version=3.5.0rc0) | [PyPI](https://pypi.org/project/scitools-iris/3.5.0rc0/) |
| `v3.5.0` | Released | [Iris v3.5.0](https://github.com/orgs/SciTools/projects/4/views/1) | `27-04-2023` | [Docs](https://scitools-iris.readthedocs.io/en/v3.5.0/) | [Release](https://github.com/SciTools/iris/releases/tag/v3.5.0) | [conda-forge](https://anaconda.org/conda-forge/iris?version=3.5.0) | [PyPI](https://pypi.org/project/scitools-iris/3.5.0/) |

**Comment by trexfeathers** [comment]
@ESadek-MO I have updated the dates to reflect our internal discussion that moved them back.
